sql/executor.go

```diff
@@ -206,8 +206,10 @@ func (e Executor) execStmt(stmt parser.Statement, params parameters, planMaker *
 						row.Values = append(row.Values, driver.Datum{
 							Payload: &driver.Datum_IntervalVal{IntervalVal: vt.Nanoseconds()},
 						})
+					case parser.DTuple:
+						return fmt.Errorf("unsupported result type: tuple, remove extra parenthesis around column %s", result.Columns[i])
```

Comment:
Agreed that the advice is inappropriate. I think we should have a specific error message for trying to return a tuple. Something like: `"unsupported tuple result"`.